cves/CVE-2013-0854.yml

```diff
@@ -64,7 +64,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This vulnerability allows for remote attackers to take advantange of a lack
+  of input validation for the decompression of MJPEG (Motion JPEG, created when each image in
+  a video is turned into a JPEG) files. Specifically, the start and end coefficient for the scan
+  that occurs when decompressing a MJPEG could have been alted in various ways such as being too 
+  large of swapped around, that would result in unintended behavior.  
```

Comment:
Good - this was a good explanation of the context. I could use a little bit more about what you mean by swapped around and what types of unintended behavior could come out of it. Or, think about it this way, if an attacker was able to develop an exploit for this, what would they be able to do?

Reply:
I do agree with Professor Meneely. The CVE score would be a great place to start.